### PR TITLE
Add semaphore config

### PR DESCRIPTION
**Pull request by anchitj**
Add semaphore config having jobs:
1. Linux x64 build and test
2. OSX x64 build and test
3. Windows x64 build and test
4. Artifacts from windows build.

Have commented integration tests for now as they are flaky.
1. Integration tests on Linux
2. Schema registry and serdes integration tests on Linux

### CODE REVIEW

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      - checkout
+  env_vars:
+    - name: CONFIGURATION
+      value: Release
+    - name: DOTNET_CLI_TELEMETRY_OPTOUT
+      value: 'true'
+
+blocks:
+  - name: 'Linux x64'
+    dependencies: []
```
Comment (PrasanthV454) [inline .semaphore/semaphore.yml:18]: Styling. Missing " " between [].

Comment (anchitj) [reply .semaphore/semaphore.yml]: Updated

**Comment by PrasanthV454** [comment]
LGTM. But you can once check this [PR] (https://github.com/confluentinc/confluent-kafka-go/pull/926) for moving out all the commons and to avoid powershell commands.

**Review by PrasanthV454 — APPROVED** [review]
LGTM. But you can once check this [PR] (https://github.com/confluentinc/confluent-kafka-go/pull/926) for moving out all the commons and to avoid powershell commands.

**Pull request merged**